code/datums/mapgen/planetary/LavaGenerator.dm

```diff
@@ -112,9 +115,9 @@
 		/mob/living/simple_animal/hostile/asteroid/goliath/beast/ancient/crystal = 1,
 		/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/forgotten = 1,
 		/mob/living/simple_animal/hostile/asteroid/hivelord/legion/crystal = 1,
-		/obj/structure/spawner/lavaland/low_threat = 12,
-		/obj/structure/spawner/lavaland/medium_threat = 4,
-		/obj/structure/spawner/lavaland/high_threat = 2,
+		/obj/structure/spawner/lavaland/low_threat = 8,
```

Comment:
Yeah, I turned down tendril weight a bit because the amount of mobs spawning entities per planet just felt a bit excessive at the previous weights and with ore veins. Though these values were more just me probing the waters with what would be better during local testing.

Reply:
Seems fine at least - easy to tweak, anyways.